checker/jdk/nullness/src/java/util/Map.java

```diff
@@ -233,7 +233,7 @@
     // collections).  Some other implementation do accept nulls and are so
```

Comment:
You probably want to remove the big comment here now (or revise it if you want to continue to call out this policy on `Map.get` specifically).